database/migrations/2016_02_27_103851_create_osu_slack_users_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateOsuSlackUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('osu_slack_users', function (Blueprint $table) {
+            $table->string('slack_id', 50);
+            $table->integer('user_id');
```

Comment:
That looks like a fucked up oversight/bug.
https://github.com/laravel/framework/issues/8840#issuecomment-182575053